runelite-client/src/main/java/net/runelite/client/plugins/opponentinfo/OpponentInfoPlugin.java

```diff
@@ -164,6 +165,43 @@ public void onGameTick(GameTick gameTick)
 		}
 	}
 
+	@Subscribe
+	public void onVarbitChanged(VarbitChanged event)
+	{
+		if ((client.getVar(Varbits.RAID_HEALTH_BAR_OVERLAY_VALUE) != lastHealth) || update ||
+			(config.showVanillaPercentages() && (event.getIndex() == 1682 || event.getIndex() == 1683)))
+		{
+			if (lastHealth == 0)
+			{
+				opponentInfoOverlay.unHideHPBar();
+			}
+			lastHealth = client.getVar(Varbits.RAID_HEALTH_BAR_OVERLAY_VALUE);
+			opponentInfoOverlay.updateRaidVars(lastHealth);
+			update = false;
+		}
+	}
+
+	@Subscribe
+	public void onConfigChanged(ConfigChanged event)
+	{
+		if (event.getGroup().equals("opponentinfo") || event.getGroup().equals("runelite"))
```

Comment:
Why is `runelite` in here? I don't really understand this at all as written.

Reply:
When the plugin itself is enabled or disabled through the config it should also remove/unhide the health bar, but I forgot I could do that through the startup and shutdown. I've removed the runelite part in the latest commit